testData/solutions/win32.json5

```diff
@@ -7117,6 +7212,7 @@
                             "title": "Keyboard as preferred input method",
                             "description": "Set the keyboard as the preferred input method",
                             "type": "boolean",
+                            // GPII-4207 This is coming out of the capture as 0 rather than false... TODO
```

Comment:
I suspect there is some leftover sloppiness around truthy and falsy values that we'll need to pin down somehow, ideally when we have "intra-application settings" and can more easily have a user-facing true/false setting that is transformed to a 1/0 for a settings-handler specific setting.